### PR TITLE
[MRG] a complete workflow for creating and updating content registry

### PR DESCRIPTION
**Pull request by ctb**
review requests:

- [ ] arthur: test execute; visual inspection that markdown is ~ok
- [x] rayna: documentation and test execute

### CODE REVIEW

**Comment by ctb** [comment]
ready for review! at least it all works in my hands which is a good start :)

I'll plan on updating documentation while waiting for feedback.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+pip install git+https://github.com/nih-cfde/cfde-deriva.git
+```
+
+Alternatively, running
+```
+pip install git+https://github.com/nih-cfde/cfde-deriva.git snakemake
+```
+should get you there.
+
+## Quickstart - building
```
Comment (raynamharris) [inline README.md:20]: Need to add steps for cloning the repo (and maybe checking out a branch)

**Review by raynamharris — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (raynamharris) [reply README.md:20]: From command line for checking out specific PRs to test locally

```
git clone https://github.com/nih-cfde/update-content-registry/
git fetch --all
git checkout -b ＜remotebranch＞ origin/＜remotebranch＞
```

then to test on app-dev 

```
python3 -m cfde_deriva.cfde_login login app-dev.nih-cfde.org
export CFDE_SUBMIT_SERVICE_INSTANCE=dev
make clean
make 
make update
```

Comment (ctb) [reply README.md:20]: I'll put the clone/fetch stuff over in #2, thanks!

In re pushing to app-dev and updating app-dev, this is only something that the CFDE-CC team will have permissions to do so we will document it internally. (There are multiple reasons for this but the simplest reason is that we can't limit it to app-dev - if you have this power on app-dev, you have this power on the live server/full portal, too!)

**Pull request merged**